### PR TITLE
[Mdct 197] 2022 Forms & PDF

### PR DESCRIPTION
**Pull request by BearHanded**
# Description

[Ticket](https://qmacbis.atlassian.net/browse/MDCT-197)

Links to the new pdf.
Forms can be generated for states for 2022.
Includes a fix for the 2022 template for jsonpath updates, and for how radio buttons display children (now they do.)
Seeds the 2022 forms into each env on deploy.

## How to test

Check out the deployed branch here: https://d3dbtfz10nfh9d.cloudfront.net/

It already has the 2022 forms generated, so feel free to click around. 
If you're really feeling generous, check out the red-lined PDF attached to the ticket and double check that differences were adjusted.

Happy to walk through this branch with anyone who cares.

## Dependencies

None

# Get it done

Now that the PR is open this is what happens next

## Code authors checklist

- [x] I have performed a self-review of my code
- [x] I have added thorough tests. [Testing Doc](https://qmacbis.atlassian.net/wiki/spaces/CM/pages/2914025525/Test+Suite+and+Testing+Research)
- [ ] Necessary analytics were added, or no new analytics were required
- [x] I have made corresponding changes to the documentation
- [x] I have assigned the PR to myself

## Reviewers Checklist (Two different people)

- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

## Assignee

- [ ] I have closed the PR after the review and necessary changes (squashing preferred)

### CODE REVIEW

**Review by karla-vm — DISMISSED** [review]
chef's kiss

**Review by karla-vm — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -53,6 +53,15 @@ If you want to a visual view of your dynamodb after the application is up and ru
 
 - to run the dynamodb gui, run `DYNAMO_ENDPOINT=http://localhost:8000 dynamodb-admin` in a new terminal window
 
+### Seed Data
+
+There are two mechanisms for seeding data.
+
+- Locally, seed data is controlled by the `database/serverless.yaml` seed section. Note the data is pulled from both the /seed and /seed-local folders.
```
Comment (gmrabian) [inline README.md:60]:
```suggestion
- Locally, seed data is controlled by the `database/serverless.yml` seed section. Note the data is pulled from both the /seed and /seed-local folders.
```

File: README.md [hunk 2]
```diff
@@ -53,6 +53,15 @@ If you want to a visual view of your dynamodb after the application is up and ru
...
+  - The seed and seed-local folders are just seperated for convention and clarity, they have no special behavior
```
Comment (gmrabian) [inline README.md:61]:
```suggestion
  - The seed and seed-local folders are just separated for convention and clarity, they have no special behavior
```

File: services/app-api/handlers/formTemplates/post.ts [hunk 3]
```diff
@@ -122,11 +122,12 @@ const saveBatch = async (tableName: string, items: any) => {
     batch.push({ PutRequest: { Item: items[i] } });
     // Submit every 25, or when you're on the last item
     if (i == items.length - 1 || batch.length == 25) {
-      dynamoDb.batchWriteItem({
+      await dynamoDb.batchWriteItem({
         RequestItems: {
           [tableName]: batch,
         },
       });
+      // eslint-disable-next-line no-console
```
Comment (gmrabian) [inline services/app-api/handlers/formTemplates/post.ts:130]: no console?

Comment (BearHanded) [reply services/app-api/handlers/formTemplates/post.ts]: good catch, relic of a painful debugging session in a deployed env

**Review by karla-vm — APPROVED** [review]

**Review by gmrabian — APPROVED** [review]
stuff is cool

**Review by braxex — APPROVED** [review]
finally a reasonably-sized PR! lol jk lgtm

**Pull request merged**